documentation/source/building.rst

```diff
@@ -173,6 +173,12 @@ Environment Variables
 
     ``COCOTB_ANSI_OUTPUT=0`` suppresses the ANSI output in the log messages
 
+.. envvar:: NO_COLOR
+    
+    From http://no-color.org : All command-line software which outputs text with ANSI color added should check for the presence 
+    of a NO_COLOR environment variable that, when present (regardless of its value), prevents the addition of ANSI color.
```

Comment:
```suggestion
    of a ``NO_COLOR`` environment variable that, when present (regardless of its value), prevents the addition of ANSI color.
```

Reply:
Thank you for your patience. I'm learning and this is my first try.